lib/src/goldens/golden_camera.dart

```diff
@@ -3,18 +3,20 @@ import 'dart:ui';
 import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import 'package:flutter_test_goldens/src/goldens/golden_rendering.dart';
+
 /// A camera for taking golden screenshots and storing them for later reference.
 class GoldenCamera {
-  GoldenCamera(this._tester);
-
-  final WidgetTester _tester;
+  GoldenCamera();
 
   List<GoldenPhoto> get photos => List.from(_photos);
   final _photos = <GoldenPhoto>[];
 
   /// Takes a screenshot of the given [finder] and stores it in [photos]
   /// along with its [description].
-  Future<void> takePhoto(Finder finder, String description) async {
+  Future<void> takePhoto(String description, [Finder? finder]) async {
```

Comment:
Please document that we look for a `GoldenImageBounds` when the `finder` isn't provided.